Source/SwiftLintFramework/Rules/MarkRule.swift

```diff
@@ -39,20 +39,112 @@ public struct MarkRule: ConfigurationProviderRule {
         ]
```

Comment:
please add `corrections` to this `RuleDescription` to cover correctable test scenarios.